db/migrate/20190529103851_add_completed_at_to_tslr_claims.rb

```diff
@@ -0,0 +1,5 @@
+class AddCompletedAtToTslrClaims < ActiveRecord::Migration[5.2]
+  def change
+    add_column :tslr_claims, :completed_at, :datetime
```

Comment:
let's add an index on this whilst we're here, I can foresee this being the thing that we use to sort claims when displaying them to administrative users.